src/Microsoft.Health.Fhir.Core/Models/SearchParameterInfo.cs

```diff
@@ -17,6 +17,7 @@ public class SearchParameterInfo : IEquatable<SearchParameterInfo>
     {
         public SearchParameterInfo(
```

Comment:
>SearchParameterInfo [](start = 15, length = 19)

Unrelated to your PR, but since you touching this file....

Can we kill this constructor?
it has 4 usages comparing to 35 of other constructor
Out of 4 usages:
1) searchParam.Type?.ToString(),
2) searchParamType: ValueSets.SearchParamType.String.ToString()

Reply:
Agreed, I have removed it.